src/UnitCell.cpp

```diff
@@ -44,7 +44,11 @@ UnitCell::UnitCell(double a, double b, double c, double alpha, double beta, doub
     beta_(beta),
     gamma_(gamma)
 {
-    if (alpha_ == 90 && beta_ == 90 && gamma_ == 90) {
+    auto is_90 = [](double angle) {
+        // We think that 89.999° is close enough to 90°
```

Comment:
Very true!

Reply:
I added a change below, to actually set them to 90 in case they were not. I think this is less surprising than having an ORTHOROMBIC cell with non-diagonal cell matrix.